src/components/PullQuote.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="pull-quote">
+    <p class="pull-quote-quote">
+      <span class="o-bg-text-accent">
+        {{ quote }}
+      </span>
+    </p>
+    <span class="pull-quote-author">{{ author }}</span>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'PullQuote',
+  props: {
+    quote: {
+      type: String,
+      default: null
+    },
+    attribution: {
```

Comment:
looks good except I think you meant this to be named "author"?

Reply:
ah yeah i was going back and forth on that (and must have missed this one) because the field on streamfield block in the cms is actually called 'attribution' but i think author is still fine here